### PR TITLE
nits

### PR DESCRIPTION
**Pull request by pcuenca**
A couple of additional notes:
- Some authors don’t belong to the `jhu-clsp` org, but have been assigned to it in the authors list.
- Would perhaps be cool to draw a couple of horizontal lines for the comparison between encoders and decoders at different sizes, which was super interesting to read.
- ~~The community will probably ask how they compare with SmolLM3 (released yesterday), but I understand it takes time and resources.~~

### CODE REVIEW

File: ettin.md [hunk 1]
```diff
@@ -22,36 +22,36 @@ authors:
   org: jhu-clsp
 ---
 
-# Seq vs Seq: the Ettin Suite of Paired Encoders and Decoders
+# Ettin Suite: SoTA Paired Encoders and Decoders
```
Comment (pcuenca) [inline ettin.md:25]: Title should ideally be in sync between the metadata, yaml, and markdown.

File: ettin.md [hunk 2]
```diff
@@ -22,36 +22,36 @@ authors:
...
 
 ## TL;DR
 
-What would happen if you took the ModernBERT recipe and applied it to a decoder-only model? Turns out, a state-of-the-art decoder language model: beating Llama 3.2 1B and SmolLM 2 models! 
+What would happen if you took the ModernBERT recipe and applied it to a decoder-only model? Turns out, a state-of-the-art decoder language model that beats Llama 3.2 1B and SmolLM2!
```
Comment (pcuenca) [inline ettin.md:29]: ~~Would be cool to compare with SmolLM3 (released yesterday).~~

On the models beating the baselines, for the 360-410M category I see Ettin being ahead of SmolLM2 in some categories and behind in others, with the same average. Perhaps we could relax and say "that beats or matches" instead (suggested something similar in the detailed discussion later on).

**Comment by tomaarsen** [comment]
Borrowing this PR a bit as well, if that's okay. Some more comments:
* I'm sure you're aware, but the image/thumbnail is not in place yet.
* The `Sizes of Ettin models` image looks to have way too much padding: https://github.com/JHU-CLSP/ettin-encoder-vs-decoder/blob/main/assets/sizes.jpg?raw=true
* `This allows us to test the effects of scale and provides a wide variety of models so you can use whether you need a blazing fast on-device model or a powerful but slower model.`: I would split this sentence up, or rewrite it somewhat, as the "wide variety of models so you can use whether ..." doesn't make sense to me.
* It's not very clear to me whether the `Encoder Results: Beating ModernBERT` table stats use extra task-specific finetuning or not, there's no details on that.

Very cool to see the difference between encoders/decoders on Retrieval. 

- Tom

**Comment by merveenoyan** [comment]
it would be great to have MNLI fine-tuned checkpoints and dedicated fine-tuning scripts for both encoder and decoder, will check once they're added :) great otherwise!
